CONTRIBUTING.md

```diff
@@ -11,6 +11,53 @@ composer install && ./vendor/bin/phpunit
 -   [Make a pull request](https://help.github.com/articles/using-pull-requests)
 -   Thanks!
 
+Testing
+=======
+
+Unit tests
+----------
+
+To run the unit tests, install the dependencies with Composer:
+
+```sh
+$ composer install
+```
+
+Then run the Composer "test" script:
+
+```sh
+$ composer test
```

Comment:
I see some skipped tests when I run this, but it's not clear why.  E.g:
```
1) Bugsnag\BugsnagBundle\Tests\Listener\BugsnagListenerTest::testOnConsoleException
ConsoleExceptionEvent class not present

4) Bugsnag\BugsnagBundle\Tests\Listener\BugsnagListenerTest::testItShouldFlushWhenAWorkerMessageHandledEventFires
This test requires symfony/messenger
```

Reply:
1) Symfony renamed the console error class so there are tests for both `ConsoleExceptionEvent` and `ConsoleErrorEvent`. Which one runs depends on the version of Symfony you install — if you just ran `composer install` then you'd get the latest v5.x so would be running the `ConsoleErrorEvent` test

2) We don't have a hard dependency on Symfony Messenger as it was introduced in Symfony 4.1 so it doesn't get installed automatically. You can manually `composer require symfony/messenger` to get that test running. CI requires it manually when the version is high enough:

https://github.com/bugsnag/bugsnag-symfony/blob/4971ab48535aa53f81f753d0212195e4951733df/.ci/require-symfony.sh#L51-L55